content/docs/command-reference/add.md

```diff
@@ -336,28 +338,21 @@ $ tree .dvc/cache
 Only the hash values of the `dir/` directory (with `.dir` file extension) and
 `file2` have been cached.
 
-## Example: Transfer to the cache
+## Example: Transfer to an external cache
 
-When you have a large dataset in an external location, you may want to add it to
-the <abbr>project</abbr> without having to copy it into the workspace. Maybe
-your local disk doesn't have enough space, but you have setup an
-[external cache](/doc/use-cases/shared-development-server#configure-the-external-shared-cache)
-that could handle it.
+Sometimes you may want to add a large dataset currently found in an external
+location, so it becomes local to the project. However, your local file system
+may not have enough space to download it — which is needed to add data in DVC,
+right? Not necessarily!
 
-The `--out` option lets you add external paths in a way that they are
+The `--out` option lets you add external data in a way that it's
 <abbr>cached</abbr> first, and then
 [linked](/doc/user-guide/large-dataset-optimization#file-link-types-for-the-dvc-cache)
-to a given path inside the <abbr>workspace<abbr>. Let's initialize an example
-DVC project to try this:
-
-```dvc
-$ mkdir example # workspace
-$ cd example
-$ git init
-$ dvc init
-```
+to a given path inside the <abbr>workspace</abbr>. Combined with an
+[external cache](/doc/use-cases/shared-development-server#configure-the-external-shared-cache)
+setup, this let's you avoid using your local file system completely.
 
-Now we can add a `data.xml` file via HTTP for example, putting it a local path
+For example, we can add a `data.xml` file via HTTP, outputting it a local path
```

Comment:
It seems a bit ambiguous, maybe use with "to"
```suggestion
For example, we can add a `data.xml` file via HTTP, outputting it to a local path
```

Reply:
Yep thanks @isidentical that's a typo!